todo-app/src/App.js

```diff
@@ -1,21 +1,107 @@
-import React, { Component } from 'react';
+import React from 'react';
 import logo from './logo.svg';
 import './App.css';
+import List from './list'
 
-class App extends Component {
-  render() {
-    return (
-      <div className="App">
-        <div className="App-header">
-          <img src={logo} className="App-logo" alt="logo" />
-          <h2>Welcome to React</h2>
+  class App extends React.Component {
+    constructor (props){
+        super(props);
+        this.state = {
+          todos: [
+               {
+                description: "Read more about React",
+                done:        true
+               } ,
+               {
+              description: "Do the homework",
+               done:        false
+              },
+            ],
+          newToDo:{
+            description: "",
+            done:        false
+           }
+        }
+
+    }
+
+    handleSubmit(event){
+      event.preventDefault()
+      const {newToDo} = this.state  
+      const todos = [...this.state.todos, newToDo]
+      this.setState({todos})
+      document.getElementById("addItem").reset();
```

Comment:
We should avoid using `document.getElementById` and try to use what React offers us in terms of event handling, data manipulation and binding, and state. I guess you are using this line of code to clear the value of the description input. However, if you turn `<input type="text"  onChange={this.handleChange.bind(this)}/>` into a [controller component](https://facebook.github.io/react/docs/forms.html#controlled-components)

```html
<input type="text" value={this.state.newTodo.description} onChange={this.handleChange.bind(this)}/>
```

you wouldn't need to do this and just resetting the state with

```javascript
this.setState({
  newToDo: {
    description: '',
    done: false
  }
});
```

would do the job! Does this make sense?